### PR TITLE
87 add admin info page

### PR DESCRIPTION
**Pull request by nathnael-desta**
Add a place where an admin can submit basic info on first login

### CODE REVIEW

File: app/layout.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
+
     <SessionWrapper>
```
Comment (misudesu) [inline app/layout.tsx:11]: SessionWrapper add in to inside body element

File: components/shared/CreateOrg/CompanyNameField.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { FormField, FormItem, FormLabel, FormControl, FormMessage } from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+
+interface CompanyNameFieldProps {
+  control: any; // Replace with the appropriate type if available
```
Comment (misudesu) [inline components/shared/CreateOrg/CompanyNameField.tsx:5]: why any?

File: components/shared/GetStarted/GetStartedAdminForm.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+"use client"
+
```
Comment (misudesu) [inline components/shared/GetStarted/GetStartedAdminForm.tsx:2]: 👌👌👌⚡

**Review by misudesu — APPROVED** [review]

**Comment by misudesu** [comment]
check the conflict.

File: app/layout.tsx (hunk 1 above)
Comment (nathnael-desta) [reply app/layout.tsx:11]: i have put it inside and commited it into the pull request

File: components/shared/CreateOrg/CompanyNameField.tsx (hunk 2 above)
Comment (nathnael-desta) [reply components/shared/CreateOrg/CompanyNameField.tsx:5]: i have removed unnecessary uses of any and commited it into the pull request

**Pull request merged**